src/main/java/net/geforcemods/securitycraft/screen/EditSecretSignScreen.java

```diff
@@ -128,7 +142,7 @@ public void render(MatrixStack matrix, int mouseX, int mouseY, float partialTick
 		int k = textInputUtil.func_216896_c();
 		int l = textInputUtil.func_216898_d();
 		int i1 = minecraft.fontRenderer.getBidiFlag() ? -1 : 1;

```

Comment:
This doesn't seem to exist in the vanilla sign anymore. Please compare this and the following code to vanilla, see what changed and apply the changes here as well.